### PR TITLE
page content schema

### PR DESCRIPTION
**Pull request by gsSuper**

### CODE REVIEW

File: artefacts/pageContent_schema.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Decentralised Identifier of the entity for which the PageContent will be made",
+          "format": "DID",
+          "examples": ["did:ixo:CYCc2xaJKrp8Yt947Nc6jdff"]
+        },
+        "@type": {
+          "type": "string",
+          "default": "ixo:page",
```
Comment (gsSuper) [inline artefacts/pageContent_schema.json:17]: @ig-shaun to confirm

File: artefacts/pageContent_schema.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                  "code",
+                  "rawHTML",
+                  "header",
+                  "quote",
+                  "checklist",
+                  "delimitter"
+                ]
+              },
+              "data": {
+                "type": "object",
```
Comment (gsSuper) [inline artefacts/pageContent_schema.json:48]: @ig-shaun 
`data` should probably be an any type as every contents has different values.

for example:
```
{
	id: "z3rRA9MDzL",
	type: "paragraph",
	data: {
		text: "Hello World!"
	}
}
```

vs

```
{
	id: "Y23Hu9teuc",
	type: "raw",
	data: {
		html: "<html>...</html>"
	}
}
```

Comment (ig-shaun) [reply artefacts/pageContent_schema.json:48]: `data` is a linkedResource that will have the pattern:
```
{
    "type": "data",
    "id": "#fwefw0wu5-w0j5-wjr",  // the content identifier (cid) of the resource
    "description": "IoT Events Data",
    "mediaType": "json",
    "endpoint": "#cellnode/fwefw0wu5-w0j5-wjr",
    "proof": "fwefw0wu5-w0j5-wjr", // matches the cid in this instance, as the content hash serves as a proof
    "encrypted": "false"
  }
```

**Comment by gsSuper** [comment]
@ig-shaun 
Just updated the schema based on your reply. Please feel free to modify it.

**Review by alwyn-ixo — APPROVED** [review]

**Pull request merged**